scripts/check_changeset_completeness.ts

```diff
@@ -21,7 +21,10 @@ const changedFiles = await gitClient.getChangedFiles(baseRef);
 const modifiedPackageDirs = new Set<string>();
 
 changedFiles
-  .filter((changedFile) => changedFile.startsWith('packages/'))
+  .filter(
+    (changedFile) =>
+      changedFile.startsWith('packages/') && !changedFile.endsWith('test.ts')
+  )
   .forEach((changedPackageFile) => {
```

Comment:
nit: just a thought. If we use a `if` condition instead of `filter` we can avoid one loop. Not sure how much performance we gain and if it worth it to do. Open to discuss.

Reply:
in our case this would fall under micro-optimizations. we wan't notice a difference.

Reply:
I agree, almost the entirety of the run time for this check is in the next step: `Validate that changeset has necessary dependency updates`